patches/minecraft/net/minecraft/world/item/enchantment/EnchantmentHelper.java.patch

```diff
@@ -38,6 +38,24 @@
        }, p_44834_);
        return mutablefloat.floatValue();
     }
+@@ -168,7 +_,7 @@
+          m_44853_(enchantmenthelper$enchantmentvisitor, p_44824_.m_20158_());
+       }
+ 
+-      if (p_44825_ instanceof Player) {
++      if (false && p_44825_ instanceof Player) { // Forge: Fix MC-248272
```

Comment:
A simpler patch would be to insert `if (false)` directly above the existing `if` statement. (Same suggestion for the other hunk in the patch.)
```suggestion
+      if (false) // FORGE: Fix MC-248272
       if (p_44825_ instanceof Player) {
```